packages/listener-solidarity/src/components/Ticket/closeTickets.ts

```diff
@@ -0,0 +1,53 @@
+import client from "../../zendesk";
+import dbg from "../../dbg";
+import { Ticket } from "../../types";
+import Bottleneck from "bottleneck";
+
+const limiter = new Bottleneck({
+  maxConcurrent: 1,
+  minTime: 1000,
+});
+
+const log = dbg.extend("closeTickets");
+
+// const userTickets = await limiter.schedule(() => listUserTickets(ticket));
+
+export default (tickets: Ticket[]) => {
+  log("Entering closeTickets");
+  const updateTickets = tickets.map(async (ticket, i: number) => {
+    const changeStatus = {
+      ticket: {
+        status: "closed" as "closed",
+        comment: {
+          body: "Fechado após MSR fazer um novo pedido de acolhimento identico",
```

Comment:
essa lógica não é justamente o contrário? só abrir ticket se o pedido de acolhimento for diferente? se for igual cria um novo e o novo é fechado?